definitions/benefit/sheets/ComplexTypes/ComplexTypes.json

```diff
@@ -544,5 +544,10 @@
   {"LiveFrom": "13/01/2025", "ID": "hearingOutcomeDetails", "ListElementCode": "completedHearings",  "FieldType": "DynamicList", "ElementLabel": "Hearings", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "18/10/2024", "ID": "hearingOutcomeValue", "ListElementCode": "hearingOutcomeId", "FieldType": "FixedList", "FieldTypeParameter": "FL_caseOutcome", "ElementLabel": "Hearing Outcome", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "18/10/2024", "ID": "hearingOutcomeValue", "ListElementCode": "didPoAttendHearing", "FieldType": "YesOrNo", "ElementLabel": "Did PO Attend?", "SecurityClassification": "PUBLIC"},
-  {"LiveFrom": "18/10/2024", "ID": "hearingOutcomeValue", "ListElementCode": "completedHearings", "FieldType": "DynamicList", "ElementLabel": "Hearing", "SecurityClassification": "PUBLIC"}
+  {"LiveFrom": "18/10/2024", "ID": "hearingOutcomeValue", "ListElementCode": "completedHearings", "FieldType": "DynamicList", "ElementLabel": "Hearing", "SecurityClassification": "PUBLIC"},
+  {"LiveFrom": "13/03/2025", "ID": "ftaRequest", "ListElementCode": "requestDueDate", "FieldType": "DateTime", "ElementLabel": "Request Due Date Time", "SecurityClassification": "PUBLIC"},
```

Comment:
change line 547 ElementLabel from "Request Due Date Time" to "FTA Comms Reply Due Date" and change FieldType from "DateTime" to "Date"

Reply:
done